specs/engage-digital_openapi3.yaml

```diff
@@ -341,6 +343,51 @@ paths:
       summary: Getting an attachment from its id
       tags:
         - Attachments
+  '/bots/handover':
+    post:
+      description: This method is used to handover a thread from bot to agent or from agent to bot

```

Comment:
could you add something like, only for messenger, see Bot Api section in documentation

Reply:
Should be better now